packages/api/src/dwn-api.ts

```diff
@@ -767,6 +771,8 @@ export class DwnApi {
 
         const { reply: { entry, status } } = agentResponse;
 
+        status.ok = (200 <= 200 && status.code <= 299);
```

Comment:
Do you know what the intention behind the `200 <= 200` check is?

Reply:
hi @shamilovtim, apologies that was a mistake on my part. it was supposed to be:

status.ok = (200 <= status.code && status.code <= 299)